tools/docker/tests/test_flows.sh

```diff
@@ -180,6 +180,48 @@ test_client_steering_mandate() {
         'grep -i -q "ACK_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
 }
 
+test_client_association_dummy(){
+    status "test client association dummy"
+    check_error=0
+
+    dbg "Connect dummy STA to wlan0"
+    check docker exec -it repeater1 sh -c \
+        'echo "AP-STA-CONNECTED 11:22:33:44:55:66" > /tmp/$USER/beerocks/wlan0/EVENT'
```

Comment:
Please make a variable for the STA address and use it below as well. Also, use a different address than in the other tests, to make sure we can distinguish the different tests in the log file.